chore(): task#0106

SNKz0610 opened the pull request:
Adding Android Homeworks 01/06/2021 about Intent.
Closes task#0106

SNKz0610 committed:
chore(): task#0106
Adding Android Homeworks 01/06/2021 about Intent.
Closes task#0106

linhvnguyen9 commented on Homeworks_25052021_DataBinding/app/src/main/java/com/snkz/databinding/MainActivity.kt:
```diff
@@ -29,6 +32,34 @@ class MainActivity : AppCompatActivity() {
+                } else {
+                    mainBinding.btnEntername.setOnClickListener {
+                        val intent : Intent = Intent(this@MainActivity, ShowNameActivity::class.java)
+                        intent.putExtra("name_user", name)
```
b nên tách `"name_user"` thành 1 hằng số compile time để tránh trường hợp bên nhận ghi sai tên và không đọc được dữ liệu

linhvnguyen9 commented on Homeworks_25052021_DataBinding/app/src/main/java/com/snkz/databinding/ShowNameActivity.kt:
```diff
@@ -0,0 +1,23 @@
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_show_name)
+        val btnBack : Button = findViewById(R.id.btn_back)
```
chỗ này b có thể xài data binding ko?

SNKz0610 replied:
All done bru. Ty for your review

SNKz0610 committed:
chore(): task#0206
Using data binding in ShowNameActivity.
Turn the key user_name into const var.
Closes task#0206

SNKz0610 committed:
Merge branch 'develop' into feature/Android_Homeworks_01062021
